setup/flex.rst

```diff
@@ -193,9 +193,9 @@ If you customize these paths, some files copied from a recipe still may contain
 references to the original path. In other words: you may need to update some things
 manually after a recipe is installed.
 
-.. _`default services.yaml file`: https://github.com/symfony/recipes/blob/master/symfony/framework-bundle/3.3/config/services.yaml
+.. _`default services.yaml file`: https://github.com/symfony/recipes/blob/master/symfony/framework-bundle/4.2/config/services.yaml
```

Comment:
i was going to ask why this was a link to 4.2 and not 5.1 like the other links, but the contents of the 5.1 file is 
```
../../4.2/config/services.yaml
```